### PR TITLE
Day4: Add Navigation 

### PR DESCRIPTION
**Pull request by luqmanmusah**
Added a Navigation Bar to the website.

Clicking on any link on the navigation Bar should not redirect to another page.

Used the library - Luxon to add the current date to the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,26 +7,60 @@
...
   </div>
   <div>
     <h2 id="second-title">Add new book:</h2>
     <br>
-      <form action="/index.html">
+      <form id="form-field" action="/index.html">
         <label for="title"></label><br>
         <input type="text" id ="title" name="title" placeholder="Tiltle"><br><br>
         <label for="author"></label><br>
         <input type="text" id="author" name="author" placeholder="Author"><br><br>
```
Comment (dasileker) [inline index.html:43]: - [ ] Please make the input field required I still can add an empty book to the list.

File: stylesheets/style.css [hunk 2]
```diff
@@ -6,6 +6,59 @@ body {
   font-family: 'Poppins', sans-serif;
 }
 
+/* Navbar style */
```
Comment (dasileker) [inline stylesheets/style.css:9]: - [ ] Please remove all the commented lines in the CSS file.

**Review by dasileker — CHANGES_REQUESTED** [review]
**Hi @Team,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- Please Add a functional Demo link to your README.md the given link is broken.
- The background color is a bit darker please make it lighter for a better view experience. 
- Please change the screenshot also so we can see what is the expected result.

## Optional suggestions

- use Color palette for better colors [color palette](https://coolors.co/)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Leandro-Barretoo** [comment]
I have fixed and applied the requested changes.
Thank you for your review.

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved✨✨

## Hello Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- Try to make the background color lighter currently is not easy to read check the link a previous reviewer shared for better use of colors

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**